bithtml.py

```diff
@@ -370,10 +371,18 @@ def run_dump_grid(db_dir, db_part, bits, output, grid_dir):
                         tmp['feature'] = 'unknown'
                         bitfeat.append(tmp)
                     else:
+
+                        if bitfeat and bitfeat[-1]['feature'] is b['feature']['feature']:
+                            i += 1
+                        else:
+                            i = 0
+
                         tmp = dict()
-                        tmp['bitname'] = b['feature']['bit'][0]
+                        tmp['bitname'] = b['feature']['bit'][i]
                         tmp['feature'] = b['feature']['feature']
+                        
```

Comment:
Can you remove trailing spaces from this line?

Reply:
Yeah sure! I changed the commit message and removed those trailing spaces.